sponsor-dapp-v2/src/components/Step1.js

```diff
@@ -1,34 +1,42 @@
-import React, { Component } from "react";
+import React, { useState } from "react";
 
 import classNames from "classnames";
+import web3 from "web3";
```

Comment:
I just did it this way since we're only using it for the `utils`. Didn't think it was worth getting drizzle involved, but happy to change if you find this confusing.